sdk/src/test/java/io/dapr/it/DaprIntegrationTestingRunner.java

```diff
@@ -0,0 +1,87 @@
+package io.dapr.it;
+
+import org.junit.Assert;
+
+import java.io.*;
+import java.util.UUID;
+import java.util.concurrent.*;
+
+
+public class DaprIntegrationTestingRunner {
+
+    private Runtime rt = Runtime.getRuntime();
+    private Process proc;
+
+    private  String successMessage;
+    private Class serviceClass;
+    private Boolean isGrpc;
+    private int port;
+    private int sleepTime;
+    private String appName;
+
+    private DaprIntegrationTestingRunner(String successMessage, Class serviceClass, Boolean isGrpc, int port, int sleepTime) {
+        this.successMessage = successMessage;
+        this.serviceClass = serviceClass;
+        this.isGrpc = isGrpc;
+        this.port = port;
+        this.sleepTime = sleepTime;
+        this.generateAppName();
+    }
+
+    public static DaprIntegrationTestingRunner createDaprIntegrationTestingRunner(String successMessage, Class serviceClass, Boolean isGrpc, int port, int sleepTime) {
+        return new DaprIntegrationTestingRunner(successMessage, serviceClass, isGrpc, port, sleepTime);
+    }
+
+
+    public void initializeDapr() throws IOException, InterruptedException, TimeoutException, ExecutionException {
+
+        File file= new File("../");
```

Comment:
No need to use this to depend on current directory location. See my next comment.

Reply:
I already remove this unneeded file path